recipes/symjit/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "symjit" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/symjit-{{ version }}.tar.gz
+  sha256: 8b5d497d0cad18930c37fa0610b38d211ad4be56f10b533484be7b64f6d09cef
+
+build:
```

Comment:
Could you please take a look at the following and include the 3rd party licenses:
https://conda-forge.org/docs/maintainer/adding_pkgs/#rust

Reply:
3rd party license for what?

Reply:
> For some languages (Go, rust, etc.), the current policy is to include all dependencies and their dependencies in the package.

The underlying rust code uses a number of dependencies pulled in through cargo, so `cargo-bundle-licenses` automatically pulls them together and includes them in the conda package.

Reply:
Ah, sorry. I'm not familiar with rust. I did not know it was including other software. I followed the instructions in the knowledgebase.